src/db/tables/email_addresses.js

```diff
@@ -54,6 +54,20 @@ async function addSubscriberUnverifiedEmailHash (user, email) {
 
 async function resetUnverifiedEmailAddress (emailAddressId) {
   const newVerificationToken = uuidv4()
+
+  // Time in ms to require between verification reset.
+  const verificationWait = 5 * 60 * 1000 // 5 minutes
+
+  const verificationRecentlyUpdated = await knex('email_addresses')
+    .select('id')
+    .whereRaw('"updated_at" > NOW() - INTERVAL \'1 MILLISECOND\' * ?', verificationWait)
```

Comment:
It's hard to find a `where` clause here that:
- looks nice
- is cross-database (doesn't use `raw`)
- handles parameterized queries
- compares only timestamps from the DB, not the app server

I ended up trading off the "looks nice" and "cross-database" in favor of the other points, which are more about safety and correctness. We obviously control `verificationWait` in this patch, but using template strings here just to make it look a little nicer seems like it could set a trap for someone thinking it'd be safe to make that variable somehow be user-controlled.

Reply:
Makes sense!